devicetypes/smartthings/smartsense-open-closed-sensor.src/smartsense-open-closed-sensor.groovy

```diff
@@ -182,5 +190,10 @@ def configure() {
 
 	// temperature minReportTime 30 seconds, maxReportTime 5 min. Reporting interval if no activity
 	// battery minReport 30 seconds, maxReportTime 6 hrs by default
-	return refresh() + zigbee.batteryConfig() + zigbee.temperatureConfig(30, 300) // send refresh cmds as part of config
+	def commands = refresh() + zigbee.batteryConfig() + zigbee.temperatureConfig(30, 300)
+	// send refresh cmds as part of config
```

Comment:
This comment should be above the line above since it's referring to the call to `refresh()`